STATS/PlaystatsHubExit.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: STATS 
+---
+
+## _PLAYSTATS_HUB_EXIT
+
+```c
+// 0x5A46ACE5C4661132 
+void _PLAYSTATS_HUB_EXIT(Any p0);
```

Comment:
This is a list of the struct hub exit from what I found.

- `hubID`: Identifies this specific instance of the social hub, shared among players within it.
- `type`: Indicates the type of social hub the player departed from (e.g., music studio, smoking room).
- `bossId1` and `bossId2`: Unique identifiers for Boss instances present.
- `playerRole`: Denotes the player's role within the hub (e.g., CEO, bodyguard, biker prospect).
- `pulled`: Indicates if the player was forcibly removed from the social hub (e.g., due to a mission starting).
- `crewId`: Identifies the Social Club Crew to which the player belongs.
- `duration`: Records the time spent within the social hub.
- `dre`: Signifies the presence of Dr. Dre within the social hub.
- `playerCount`: Reflects the number of players present within the social hub at the time of exiting.